Library/Formula/apache-spark.rb

```diff
@@ -1,13 +1,20 @@
 class ApacheSpark < Formula
   desc "Engine for large-scale data processing"
   homepage "https://spark.apache.org/"
-  url "https://www.apache.org/dyn/closer.lua?path=spark/spark-1.6.0/spark-1.6.0-bin-hadoop2.6.tgz"
   version "1.6.0"
-  sha256 "439fe7793e0725492d3d36448adcd1db38f438dd1392bffd556b58bb9a3a2601"
   head "https://github.com/apache/spark.git"
-
   bottle :unneeded
 
+  option "without-hadoop", "Use the -without-hadoop distribution of Spark (assumes you have a local installation of hadoop)"  
```

Comment:
I had a feeling that would be the case. Perhaps it's better to split it off into a apache-spark-without-hadoop formula? I figured this scenario wouldn't affect most users (as there is invariably some manual config editing involved when running the "user-provided hadoop" version of Spark), and really only thought it was worth submitting for the people who'd like a homebrew-managed Spark install.

I could alternatively make a new formula that installs the -without-hadoop dist by default, depends on Homebrew's on `hadoop` formula, and does the configuration changes necessary to integrate it. However that *might* involve editing the Hadoop formula as well, as it doesn't seem to link its `/etc` or `/usr/local/share` files into any central version-agnostic directory. Although it's not strictly
necessary to have them linked for a functional Spark + Hadoop setup, it *does* allow you to use nice things like Hadoop extras (such as S3 and Azure Blob HDFS backends) that aren't in the default `$(hadoop classpath)`